### DISCUSSION TITLE
求 打包 博德之门2：增强版

### DISCUSSION THREAD

**Discussion by github342655596** [Gamuxorg/bbs#252] [投递和请求游戏]
游戏名称：博德之门2：增强版
游戏名称(英文)：Baldurs Gate II Enhanced Edition
下载地址 :（如果有的话）：站里面的那个没打包，安装了也不能玩，如果可以希望更新到最新版本。

**Comment by wangling12** [comment]
抽空我打包一下

**Comment by shouhuanxiaoji** [comment]
haole
